client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -195,6 +195,7 @@ class TransferDomainStep extends React.Component {
 		return (
 			<TransferDomainPrecheck
 				domain={ this.state.domain }
+				selectedSite={ this.props.selectedSite }
```

Comment:
We only need slug here.

Reply:
And....it looks like I forgot to pass this into the new component too. Will fix.